packages/owc-dev-server/README.md

```diff
@@ -33,26 +33,45 @@ Please note that the owc-dev-server *only resolves bare module specifiers*, and
 :::
 
 
-### Usage
+## Usage
+Before using owc-dev-server in your project, you need to install and add it to your package's `devDependencies`:
 ```bash
 npm i -D owc-dev-server
+# or
+yarn add --dev owc-dev-server
 ```
 
-#### Execute
-```
+Note: The owc-dev-server package is *named after the command* it provides. That's why it doesn't follow the naming pattern for the other open-wc packages and does not use a `@open-wc/` scope prefix.
+
+### Execute as script
+For convenience (i.e. to always use the same [command line options](#command-line-options)) you should add an entry to your `scripts`:
+```javascript
 // package.json
 "scripts": {
   "start": "owc-dev-server"
 }
+```
 
-// bash
+This allows you to execute owc-dev-server like this:
+```bash
+# run package.json script 'start'
 npm run start
+# or
+yarn run start
+```
 
-// or via npx
+### Execute with npx
+You can also start owc-dev-server through `npx`. This allows you to easily try out different [command line options](#command-line-options).
+```bash
+# run ad-hoc via npx
 npx owc-dev-server
 ```
 
-#### Static files
+::: warning
+If you didn't install owc-dev-server locally (as a dependency, [see above](#usage)), running `npx owc-dev-server` will start owc-dev-server, but it **will not work** properly and report errors. See [this issue](https://github.com/open-wc/open-wc/issues/366) for details.
```

Comment:
can we rephrase that? maybe something like this
```suggestion
`owv-dev-server` only works properly when installed locally. 
```

coming to think of it maybe we should remove all the `npx` stuff and e.g. better explain how to add a script and then only use `npm run start` or so - what do you think?

Reply:
Good point. Should definitely get some thought...

After all, what I loved so much about the open-wc docs, was getting clear recommendations how to set things up. And I learned a lot from the "manual setup" parts of the documentation, too. Ever since I have a better grasp on how these tools interact. (There's still much to learn for me, though.)

So... how about I give it another shot and remove all mentions of npx?

Reply:
sounds good too me :hugs:  @lars do you agree?